README.md

```diff
@@ -1,3 +1,44 @@
-# NWB-Packaging-Ophys-Base-Capsule
+# AIND Ophys NWB
 
-Used to append various ophys data to an NWB file. Attach the raw, processed, and NWB assets for a session to use. 
\ No newline at end of file
+A Python-based tool for working with ophys data in the Neurodata Without Borders (NWB) format. This library provides functionality to load and examine NWB files containing optical physiology data.
+
+## Overview
+
+This repository contains utilities for working with ophys (optical physiology) data stored in the NWB format. It allows users to load, examine, and process NWB files that contain multiplane ophys recordings. It is meant to be used in a Code Ocean workstation where processed ophys assets can be attached to the capsule and a reproducible run will produce a complete, ophys NWB.
```

Comment:
This repository extracts processed data from single plane and multiplane optical physiology datasets and packages the data into NWB format. It is meant to be used in a Code Ocean workstation....